tfjs-backend-webgpu/src/lrn_webgpu.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {util} from '@tensorflow/tfjs-core';
+import {getMainHeaderString as main, WebGPUProgram} from './webgpu_program';
+import {computeDispatch, flatDispatchLayout} from './webgpu_util';
+
+const powOperatorSnippet = `
+  var powValue = 0.0;
+  let basis = uniforms.bias + uniforms.alpha * sum;
+  if (uniforms.beta == 0.5) {
+    powValue = inverseSqrt(basis);
+  } else if (uniforms.beta == 1.0) {
+    powValue = 1.0 / basis;
+  } else {
+    powValue = exp(log(basis) * (-uniforms.beta));
+}
+`;
```

Comment:
These conditions should be moved outside of the shader.

Reply:
Per offline discussion, we can keep this to minimize the shader variants.

Reply:
Done